### PR TITLE
Better

### PR DESCRIPTION
**Pull request by reshke**
unify log messages, use ID() iface everywhere, replace client/shard id type (string->uint)

### CODE REVIEW

File: balancer/pkg/console.go [hunk 1]
```diff
@@ -185,8 +181,7 @@ func (c *Console) ProcessQuery(ctx context.Context, q string, cl client.Client)
 		err := c.coord.lockKeyRange(keyRange)
 		if err != nil {
 			spqrlog.Zero.Error().
-				Err(err).
-				Str("client", cl.ID()).
+				Err(err).Uint("client", cl.ID()).
```
Comment (Denchick) [inline balancer/pkg/console.go:184]: New line here

File: balancer/pkg/console.go [hunk 2]
```diff
@@ -199,8 +194,7 @@ func (c *Console) ProcessQuery(ctx context.Context, q string, cl client.Client)
 		err := c.coord.unlockKeyRange(keyRange)
 		if err != nil {
 			spqrlog.Zero.Error().
-				Err(err).
-				Str("client", cl.ID()).
+				Err(err).Uint("client", cl.ID()).
```
Comment (Denchick) [inline balancer/pkg/console.go:197]: New line

File: pkg/clientinteractor/interactor.go [hunk 3]
```diff
@@ -383,7 +383,7 @@ type ClientDesc struct {
 
 func (_ ClientDesc) GetRow(cl client.Client, hostname string, rAddr string) []string {
 	quantiles := statistics.GetQuantiles()
-	rowData := []string{cl.ID(), cl.Usr(), cl.DB(), hostname, rAddr}
+	rowData := []string{fmt.Sprintf("%d", cl.ID()), cl.Usr(), cl.DB(), hostname, rAddr}
```
Comment (Denchick) [inline pkg/clientinteractor/interactor.go:386]: This is slow

File: pkg/clientinteractor/interactor.go [hunk 4]
```diff
@@ -796,7 +796,7 @@ func (pi *PSQLInteractor) BackendConnections(ctx context.Context, shs []shard.Sh
 			router = s.Router()
 		}
 
-		if err := pi.WriteDataRow(sh.ID(), router, sh.ShardKeyName(), sh.InstanceHostname(), sh.Usr(), sh.DB(), strconv.FormatInt(sh.Sync(), 10), strconv.FormatInt(sh.TxServed(), 10), sh.TxStatus().String()); err != nil {
+		if err := pi.WriteDataRow(fmt.Sprintf("%d", sh.ID()), router, sh.ShardKeyName(), sh.InstanceHostname(), sh.Usr(), sh.DB(), strconv.FormatInt(sh.Sync(), 10), strconv.FormatInt(sh.TxServed(), 10), sh.TxStatus().String()); err != nil {
```
Comment (Denchick) [inline pkg/clientinteractor/interactor.go:799]: This is slow

File: pkg/datashard/datashard.go [hunk 5]
```diff
@@ -198,8 +198,6 @@ func NewShard(
 		mp:       map[uint64]shard.PreparedStatementDescriptor{},
 	}
 
-	dtSh.id = fmt.Sprintf("%p", dtSh)
```
Comment (Denchick) [inline pkg/datashard/datashard.go:201]: Nice

**Review by Denchick — CHANGES_REQUESTED** [review]

File: pkg/clientinteractor/interactor.go (hunk 3 above)
Comment (reshke) [reply pkg/clientinteractor/interactor.go:386]: We cant do much here, plus this is not hot code path

File: pkg/clientinteractor/interactor.go (hunk 4 above)
Comment (reshke) [reply pkg/clientinteractor/interactor.go:799]: same

**Review by Denchick — APPROVED** [review]

**Pull request merged**